src/util/const.ts

```diff
@@ -7,6 +7,9 @@ export const isBrowser =
   typeof navigator !== 'undefined' &&
   typeof document !== 'undefined';
 
+export const isStrictEqual: TPredicate = <T>(prev: T, next: any): next is typeof prev =>
```

Comment:
Leave it as `(a: any, b: any): boolean => a === b`
Typewise it is way more versatile and can be used in other code, but still math predicate type